### DISCUSSION TITLE
Custom wake words not on pre-made list

### DISCUSSION THREAD

**Discussion by Alfamari** [KoljaB/RealtimeSTT#8] [Q&A] [answered]
Is it possible to create custom wake words that are not on the pre-created list? If it isn't is there a reason for that?

**Comment by KoljaB** [comment, marked as answer]
I use pvporcupine for wake words as it is the most performant lib out there as of my knowledge. Unfortunately they went from fully open source to a paid model with version 1.9.5 being the last free one.  V1.9.5 only supports the wakewords from the list. 
So it is possible to use other wake words by updating pcporcupine to the latest version, but you'd need to pay for their service and it would require an online connection.

**Reply by Alfamari** [reply]
Ah, I see, thank you for the explanation and fast reply.
